### ISSUE TITLE
[Errno 111] Connection refused

### ISSUE THREAD

**Issue by chewy74** [Flexget/Flexget#2876]
### Expected behaviour:
Flexget commands to execute 

### Actual behaviour:
Each command returns error: [Errno 111] Connection refused
### Steps to reproduce:
- Step 1: ...enter any flexget command 

#### Config:
https://pastebin.com/2RETF2cB
  
#### Log:
<details>
 <summary>(click to expand)</summary>

```
(flexget) pi@pi4:~ $ flexget --debug daemon start -d     2021-03-05 11:56:57 DEBUG    manager                       Figuring out config load paths
2021-03-05 11:56:57 DEBUG    manager                       Adding virtualenv path
2021-03-05 11:56:57 DEBUG    manager                       Found config: /home/pi/flexget/config.yml
2021-03-05 11:56:57 DEBUG    manager                       Config file /home/pi/flexget/config.yml selected
2021-03-05 11:56:57 DEBUG    manager                       sys.defaultencoding: utf-8
2021-03-05 11:56:57 DEBUG    manager                       sys.getfilesystemencoding: utf-8
2021-03-05 11:56:57 DEBUG    manager                       flexget detected io encoding: utf-8
2021-03-05 11:56:57 DEBUG    manager                       os.path.supports_unicode_filenames: False
There is a FlexGet process already running for this config, sending execution there.
2021-03-05 11:56:57 DEBUG    manager                       Sending command to running FlexGet process: ['--debug', 'daemon', 'start', '-d']
Traceback (most recent call last):
  File "/home/pi/flexget/lib/python3.7/site-packages/flexget/__init__.py", line 44, in main
    manager.start()
  File "/home/pi/flexget/lib/python3.7/site-packages/flexget/manager.py", line 360, in start
    client = IPCClient(ipc_info['port'], ipc_info['password'])
  File "/home/pi/flexget/lib/python3.7/site-packages/flexget/ipc.py", line 186, in __init__
    channel = rpyc.Channel(rpyc.SocketStream.connect('127.0.0.1', port))
  File "/home/pi/flexget/lib/python3.7/site-packages/rpyc/core/stream.py", line 172, in connect
    return cls(cls._connect(host, port, **kwargs))
  File "/home/pi/flexget/lib/python3.7/site-packages/rpyc/core/stream.py", line 124, in _connect
    s = socket_backoff_connect(family, socktype, proto, sockaddr, timeout, attempts)
  File "/home/pi/flexget/lib/python3.7/site-packages/rpyc/lib/__init__.py", line 131, in socket_backoff_connect
    sock.connect(addr)
ConnectionRefusedError: [Errno 111] Connection refused
(flexget) pi@pi4:~ $
```
</details>

### Additional information:

- FlexGet version: 3.1.105
- Python version: 3.7
- Installation method: pip
- Using daemon (yes/no): yes
- OS and version: raspbian buster 
- Link to crash log:

**Comment by luizoti** [comment]
If exist, try delete this file on config diretory: `~/.flexget/.config-lock`

**Comment by chewy74** [comment]
@luizoti I rebooted the whole system and even stopped and restarted the daemon. So should the config-lock still be deleted?

Will the daemon remain active without it?

Thanks

**Comment by luizoti** [comment]
Yes, you need.

**Comment by luizoti** [comment]
This file ia created on flexger execution, with this um note poassible tô run tow instances, problaby os yor case

**Comment by chewy74** [comment]
Resolved with 3.1.106
